docs/en/04_Changelogs/beta/4.10.0-beta1.md

```diff
@@ -0,0 +1,55 @@
+# 4.10.0 beta1
+
+## Overview
+
+- [Stable module forcing installation of unstable framework](#unstable-framework)
+- [Dropping support for PHP 7.1 and PHP 7.2](#phpeol)
+- [Features and enhancements](#features-and-enhancements)
+- [Bugfixes](#bugfixes)
+
+## Stable module forcing installation of unstable framework{#unstable-framework}
+
+If you recently updated your project's `composer.json` file, you may have accidentally got an unstable version of `silverstripe/framework`.
+
+In the process of publishing Silverstripe CMS Recipe 4.10.0-beta1, we ended up releasing a few modules as "stable" but with a requirement on the unstable `silverstripe/framework` 4.10. For most projects, this won't be a ploblem if your `composer.json` file uses the `"prefer-stable": true` flag. However, if you explicitely require the very latest version of a module that depends on `silverstripe/framework ^4.10`, composer will install an unstable version of `silverstripe/framework`.
+
+To double check what version of `silverstripe/framework` is installed on your project, run this command `composer show silverstripe/framework`. This will show the currently installed version of `silverstripe/framework`. If it comes out with `4.10.0-beta1` or `4.x-dev` or `4.10-x-dev`, you are running an unstable version.
+
+To remedy this problem you can explicitely require `"silverstripe/framework": "~4.9.0"` in your composer file and run `composer update -W`. If you get conflicts, you might need to loosen some constraints to get older versions of some modules. Once you are ready to upgrade to `silverstripe/framework` 4.10.0 stable, you'll need to remove or update that explicit constraint.
```

Comment:
```suggestion
To remedy this problem you can explicitely require `"silverstripe/framework": "~4.9.1"` in your composer file and run `composer update -W`. If you get conflicts, you might need to loosen some constraints to get older versions of some modules. Once you are ready to upgrade to `silverstripe/framework` `4.10.0` stable then you can remove this explicit constraint.
```

Reply:
I think "you'll need to remove" is more precise here because can implies that it's optional.